lib/plugins/aws/package/index.js

```diff
@@ -44,6 +46,7 @@ class AwsPackage {
                   'mergeCustomProviderResources',
                   'stripNullPropsFromTemplateResources',
                   'saveServiceState',
+                  'addExportNameForOutputs',
```

Comment:
Let's move it before `saveServiceState`

Reply:
or before `mergeCustomProviderResources`, depending on the outcome of the discussion